### DISCUSSION TITLE
Is there anyone had tried the new 1.12.7 version

### DISCUSSION THREAD

**Discussion by hantsy** [mockk/mockk#908] [Q&A] [answered]
I have just updated the mockk version to 1.12.7, but all Mockk APIs are not found, when I opened the jar in the locale Maven repository(~/.m2/repository), and found the mockk.jar does not contain the class files, only include a META-INF folder.

![005](https://user-images.githubusercontent.com/854520/187108003-e18d256a-24c0-4bcf-9569-99a097d0e938.png)

I have checked the online version, https://repo1.maven.org/maven2/io/mockk/mockk/1.12.7/ it also does not contains class files.

**Comment by jarledb** [comment]
Seems broken for me as well, version 1.12.5 should be working as normal

**Reply by hantsy** [reply]
But 1.12.5 does not fix https://github.com/mockk/mockk/issues/832

**Comment by Raibaz** [comment, marked as answer]
We had an issue with dependency management when migrating to the new build flow: if you are using maven you can switch your dependency artifactId from `mockk` to `mockk-jvm` to upgrade to 1.12.7, or wait until we release 1.12.8 which should come soon and fix this issue.

Sorry for the inconvenience.

**Reply by hantsy** [reply]
Thanks, but I still encountered [issue 832](https://github.com/mockk/mockk/issues/832#issuecomment-1229954666).
